frontend/src/pages/admin/index.svelte

```diff
@@ -486,6 +810,410 @@
 				{/if}
 			</div>
 		</div>
+		<div
+			class="mx-2 my-4 col-span-12 grid grid-cols-12 gap-4 {activeView ==
+			2
+				? 'block'
+				: 'hidden'}"
+		>
+			{#key activeView}
+				<div class="col-span-12 flex flex-wrap justify-between gap-y-2">
+					<h1 class="text-3xl font-bold">Reports & Statistics</h1>
+					<div class="flex gap-x-4 items-end flex-wrap">
+						<div>
+							<label class="label" for="">
+								<span class="label-text">Date range</span>
+								<span class="label-text pr-0.5">Active</span>
```

Comment:
This should not be an option. The default data is for one month, not infinite date range. Unchecking this box just sets it to one month of data, which means there still is a date filter just a different one.

Reply:
> The default data is for one month, not infinite date range.  

This is not always true. 
When, by default, the date range is not applied, the data returned uses the server's default ranges, which are different for different data sets. Applying a certain range lets the user control the range the server applies.  

For example, the "Kitcoin sent/spent per day" graphs default to displaying the previous month on the server side because it makes sense to show that and it is also not possible to display all time within a graph. 
However, for "Kitcoin sent by staff" and "Top transactions", the server does not set a default range and returns the data for all time because it is more useful to look at trends for that data over time than for the past month. 
Letting the server choose defaults allows more useful data to be shown without user interaction, but this behavior can be changed with a click if they want something else.

I think that because the server does this, it is useful to let the user chose between what they want to see. Technically we could remove the date filter entirely and the server would still use its defaults anyways, so the toggle is there to make this more clear.

Do you think there's a better way to do this or do you still want it gone?

Reply:
Ah ok, I'm sorry I didn't understand this. I just think we should make it clear (maybe upon hover) that the kitcoin sent per day isn't the same range as everything else because it is a bit confusing.

Reply:
Is this better? 
![image](https://user-images.githubusercontent.com/37093293/174390975-178c7019-e610-4db4-bc54-a9a39ad5525f.png)

Reply:
Looks good to me!